USE-CASES/smart-agriculture.md

```diff
@@ -0,0 +1,55 @@
+## Title: Smart Agriculture (Greenhouse Horticulture)
+
+### Submitter(s): 
+
+Ryuichi Matsukura, Takuki Kamiya
+
+### Reviewer(s):
+
+### Tracker Issue ID:
+
+### Category:
+
+### Class: 
+
+### Status: 
+
+### Target Users
+
+Agricultural corporation, Farmer, Manufacturers (Sensor, other facilities), Cloud provider
+
+### Motivation:
+
+Greenhouse Horticulture controlled by computers can create an optimal environment for growing plants. This enables to improve productivity and ensure stable vegetable production throughout the year, independent of the weather. This is the result of research on the growth of plants in the 1980s. For example, in tomatoes, switching to hydroponics and optimizing the temperature, humidity and CO2 concentration required for photosynthesis resulted in a five times increase in yield. The growth conditions for other vegetables also have been investigated, and this control system is applied now.
+
+### Expected Devices:
+
+Sensors ( temperature, humidity, brightness, UV brightness, air pressure, and CO2)
```

Comment:
Other sensors and devices that we are employing in our agriculture project are: 
- Multispectral cameras that are used to extract the Leaf Area Index parameter of a crop
- Drones that survey the area. Also, they might be used as actuators (i.e. spray pesticide ) 
- Remote-controlled sprinklers
- flow meters to control the amount of water fed to the crop
- Soil moisture, conductivity and temperature
- Automatic weirs to control the flow in agriculture channels and canals
- Water level sensors (again used for channels and canals)

Reply:
@relu91, thank you for your useful comments and very sorry for my late reply. 
I agree many sensors and actuators are used in the agricultural fields. However, there are several types of ICT controlled agriculture like plant factories, Greenhouses, and open field cultivations. They have different requirements for sensors and actuators.
This use case description is for greenhouse horticulture. so, how about adding some of them you mentioned used for the greenhouse to this description?

I think the following sensors are for them;
- Multispectral camera
- Soil moisture, conductivity and temperature

Other sensors and actuators could be used mainly for open fields. Creating another use case for agriculture might be better. What do you think?